tf/chunkparser.py

```diff
@@ -278,24 +388,36 @@ def convert_v5_to_tuple(self, content):
                  self.flat_planes[1]
 
         assert len(planes) == ((8 * 13 * 1 + 8 * 1 * 1) * 8 * 8 * 4)
-        winner = float(winner)
-        assert winner == 1.0 or winner == -1.0 or winner == 0.0
-        winner = struct.pack('fff', winner == 1.0, winner == 0.0,
+
+        # jhorthos query - pls check this
+        if version == V6_VERSION:
```

Comment:
This could be `if ver == 6:` to avoid having to extract 'version' at all if you follow the suggestion above.

Reply:
okay i get all this up to here. mostly struggling to figure out git now. :-)

Reply:
fixed